UnitsNet/Scripts/UnitDefinitions/Flow.json

```diff
@@ -60,16 +60,42 @@
       ]
     },
     {
+      "SingularName": "LitersPerHour",
+      "PluralName": "LitersPerHour",
+      "FromUnitToBaseFunc": "x/3600000.00000",
+      "FromBaseToUnitFunc": "x*3600000.00000",
+      "Localization": [
+        {
+          "Culture": "en-US",
+          "Abbreviations": [ "LPH" ],
+          "AbbreviationsWithPrefixes": [ "nLPM", "μLPM", "mLPM", "cLPM", "dLPM", "kLPM" ]
```

Comment:
You can remove all occurrences of `AbberivationsWithPrefixes` property here and below, it is only needed if you need something else than the standard of prefixing the letters in front of the first abbrevation in `Abbreviations`.